### DISCUSSION TITLE
Airflow is very slow after upgrade to 2.2.3 to 2.6

### DISCUSSION THREAD

**Discussion by HanumanthaRAON** [apache/airflow#33646] [Q&A]
Hi Team,

We have recently upgraded airflow version from 2.2.3 to 2.6 and we are seeing airflow UI is responding very slowly it is taking almost 4-5 sec respond. can someone please help us to resolve the issue

**Comment by potiuk** [comment]
Likely you need more resources. You need to monitor your Airflow and see what resources are used. My best guess is that you need to increase memory. But this is absolutely wild guess. The only good answer you can get if you take a look at your deployment/monitoring and see what the problem is.
